toqito/channels/tests/test_partial_trace.py

```diff
@@ -466,742 +248,132 @@ def test_partial_trace_16_by_16_subsystems_4_2_2():
-    bool_mat = np.isclose(expected_pt_1_2, pt_1_2)
-    np.testing.assert_equal(np.all(bool_mat), True)
-
-    # Trace out the first and third subsystems:
-    pt_1_3 = partial_trace(test_input_mat, [0, 2], [4, 2, 2])
-    expected_pt_1_3 = np.array([[892, 908], [1148, 1164]])
-    bool_mat = np.isclose(expected_pt_1_3, pt_1_3)
-    np.testing.assert_equal(np.all(bool_mat), True)
-
-    # Trace out the second and third subsystems:
-    pt_2_3 = partial_trace(test_input_mat, [1, 2], [4, 2, 2])
-    expected_pt_2_3 = np.array(
+    ),[2], [4, 2, 2]),
+    # 16-by-16 matrix for subsystems 4 x 2 : trace out first and second subsystem
+    (test_input_mat5, np.array([[960, 968], [1088, 1096]]),[0, 1], [4, 2, 2]),
+    # 16-by-16 matrix for subsystems 4 x 2 : trace out first and third subsystem
+    (test_input_mat5, np.array([[892, 908], [1148, 1164]]),[0, 2], [4, 2, 2]),
+    # 16-by-16 matrix for subsystems 4 x 2 : trace out second and third subsystem
+    (test_input_mat5, np.array(
         [[106, 122, 138, 154], [362, 378, 394, 410], [618, 634, 650, 666], [874, 890, 906, 922]]
-    )
-    bool_mat = np.isclose(expected_pt_2_3, pt_2_3)
-    np.testing.assert_equal(np.all(bool_mat), True)
-
-
-def test_partial_trace_16_by_16_subsystems_4_4():
-    """Test for 16-by-16 matrix for subsystems 4 x 4."""
-    test_input_mat = np.arange(1, 257).reshape(16, 16)
-
-    # Trace out first subsystem:
-    pt_1 = partial_trace(test_input_mat, [0], [4, 4])
-    expected_pt_1 = np.array(
+    ),[1, 2], [4, 2, 2]),
+    # 16-by-16 matrix for subsystems 4 x 4 : trace out first subsystem
+    (test_input_mat5, np.array(
         [[412, 416, 420, 424], [476, 480, 484, 488], [540, 544, 548, 552], [604, 608, 612, 616]]
-    )
-    bool_mat = np.isclose(expected_pt_1, pt_1)
-    np.testing.assert_equal(np.all(bool_mat), True)
-
-    # Trace out second subsystem:
-    pt_2 = partial_trace(test_input_mat, [1], [4, 4])
-    expected_pt_2 = np.array(
+    ),[0], [4, 4]),
+    # 16-by-16 matrix for subsystems 4 x 4 : trace out second subsystem
+    (test_input_mat5, np.array(
         [[106, 122, 138, 154], [362, 378, 394, 410], [618, 634, 650, 666], [874, 890, 906, 922]]
-    )
-    bool_mat = np.isclose(expected_pt_2, pt_2)
-    np.testing.assert_equal(np.all(bool_mat), True)
-
-
-def test_partial_trace_64_by_64_subsystems_4_4_2_2():
-    """Test for 64-by-64 matrix for subsystems 4 x 4 x 2 x 2."""
-    test_input_mat = np.arange(1, 4097).reshape(64, 64)
-
+    ),[1], [4, 4]),
+    # To Do : 64-by-64 matrix for subsystems 4 x 4 x 2 x 2
     # Trace out first subsystem:
     # Trace out second subsystem:
     # Trace out third subsystem:
     # Trace out fourth subsystem:
-
     # Trace out first and second subsystem:
     # Trace out first and third subsystem:
     # Trace out first and fourth subsystem:
-
     # Trace out second and third subsystem:
     # Trace out second and fourth subsystem:
-
     # Trace out third and fourth subsystem:
```

Comment:
I kept these lines the way they are. I don't really think these are that urgent compared to other things on my list. 

It would be better to create an issue for another contributor who might be interested in `toqito`.